document-portal/file-transfer.c

```diff
@@ -566,14 +566,17 @@ stop_file_transfers_in_thread_func (GTask        *task,
         }
     }
   G_UNLOCK (transfers);
+  g_task_return_boolean (task, TRUE);
 }
 
 void
 stop_file_transfers_for_sender (const char *sender)
 {
   GTask *task;
+  g_autofree char *name = g_strdup_printf("stop_file_transfers_for_sender(%s)", sender);
 
   task = g_task_new (NULL, NULL, NULL, NULL);
+  g_task_set_name (task, name);
```

Comment:
In addition to splitting this in a separate commit, it would be lovely to have the GTask set a source tag, and assert it in `stop_file_transfers_in_thread_func` as well

Reply:
I have removed them from this PR.